scripts/utils/migrate/fetch-attribute-definitions.js

```diff
@@ -0,0 +1,14 @@
+const docsApi = require('./docs-api');
+
+const prop = (name) => (obj) => obj[name];
```

Comment:
1. Love this
2. Looks like we define this function here and in `fetch-doc-count.js`. Maybe we should make a util file for functional helpers?

Reply:
Thats probably a good idea!

Reply:
Addressed with d4f5187